pyne/alara.py

```diff
@@ -704,3 +727,24 @@ def cram(N, t, n_0, order):
         msg = 'Rational approximation of degree {0} is not supported.'.format(order)
         raise ValueError(msg)
 
+def _output_flux(ve, tag_flux,output,start,stop,direction):
+    """
+    This function is used to write netron flux into fluxin
+    :param ve: entity
+    :param tag_flux:
+    :param output: string
+    :return: string
+    """
```

Comment:
Please write this docstring in numpydoc format, so that it conforms to the other docstrings throughout pyne. Thanks!

Reply:
I rewrote this docstring in numpydoc format.